### PR TITLE
Updated readme docs

### PR DESCRIPTION
**Pull request by saratchandrakarumuri**
* Updated api entities according to usage in swift 3.0
* Updated default values

### CODE REVIEW

**Comment by dg0yal** [comment]
@singhgag have you looked this over? The biggest question in my mind is whether to change the function references to use swift syntax rather than obj-c. This PR muddles the syntax - its neither objc nor swift. What's your opinion?

**Comment by singhgag** [comment]
@dg0yal I have not. But having looked at it now, I think we are missing the `(``)` brackets. Like for example [here](https://developer.apple.com/reference/corelocation/cllocation/1423666-init). Once we fix that, I feel it should be good.

@saratchandrakarumuri Can you take a look at the Apple's API Reference as an example? Use the link I have posted above to get started.

**Comment by saratchandrakarumuri** [comment]
@dg0yal @singhgag Updated the syntax of functional references to reflect SWIFT syntax as per the Apple's API reference

File: arcgis-ios-sdk-samples/Layers/Export tiles/README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Pan and zoom into the desired area, making sure the area is within the red block
 
 ##How it works
 
-The sample uses the `exportTileCacheParametersWithAreaOfInterest:minScale:maxScale:completion:` method on `AGSExportTileCacheTask` class to generate `AGSExportTileCacheParameters` parameters by providing the area of interest and the min max scale for the tpk. It then uses these parameters in the `exportTileCacheJobWithParameters:downloadFilePath:` method to generate an `AGSJob`. The job on successful completion gives an `AGSTileCache` object which is used to create an `AGSArcGISTiledLayer` and shown in a map as preview.
+The sample uses the `exportTileCacheParameters(withAreaOfInterest:minScale:maxScale:completion:)` method on `AGSExportTileCacheTask` class to generate `AGSExportTileCacheParameters` parameters by providing the area of interest and the min max scale for the tpk. It then uses these parameters in the `exportTileCacheJob(withParams:downloadFilePath:)` method to generate an `AGSJob`. The job on successful completion gives an `AGSTileCache` object which is used to create an `AGSArcGISTiledLayer` and shown in a map as preview.
```
Comment (dg0yal) [inline arcgis-ios-sdk-samples/Layers/Export tiles/README.md]: it should be exportTileCacheJob(with:downloadFilePath:)

**Comment by dg0yal** [comment]
@saratchandrakarumuri Nice work. Just the one comment.
Once you resolve that this can be merged.

**Comment by saratchandrakarumuri** [comment]
@dg0yal Fixed the missed change.

**Pull request merged**